models/standardized_models/hubspot__marketing_events_standardized.sql

```diff
@@ -0,0 +1,43 @@
+{{ config(enabled=(var('hubspot__standardized_marketing_model_enabled', False)
+    and fivetran_utils.enabled_vars(['hubspot_marketing_enabled','hubspot_email_event_enabled','hubspot_email_event_sent_enabled'])))
+}}
+
+with events as (
+
+    select *
+    from {{ ref('hubspot__email_sends') }} 
+),
+
+standardized as (
+
+    select 
+        event_id,
+        coalesce(email_send_timestamp, created_timestamp) as activity_timestamp,
+        email_campaign_id as campaign_id,
+        recipient_email_address as email,
+        contact_id,
+        'email send' as event_type,
+        cast(null as {{ dbt.type_numeric() }}) as event_value,    
+        cast(null as {{ dbt.type_timestamp() }}) as updated_at,
+        email_subject,
+        bounces as total_bounces,
+        clicks as total_clicks,
+        deferrals as total_deferrals,
+        deliveries as total_deliveries,
+        drops as total_drops,
+        forwards as total_forwards,
+        opens as total_opens,
+        spam_reports as total_spam_reports,
+        cast(null as {{ dbt.type_int() }}) as total_unsubscribes,
```

Comment:
Same comment about why these are not able to be used if it seems they are available in the underlying table.

![image](https://github.com/user-attachments/assets/5fd0896c-b369-4551-8820-eee4b1675d2f)

Reply:
See the above comment.